src/modules/uavcan/uavcan_servers.hpp

```diff
@@ -88,7 +95,7 @@ class UavcanServers
 	static constexpr unsigned QueuePoolSize =
 		(NumIfaces * uavcan::MemPoolBlockSize * MaxCanFramesPerTransfer);
 
-	static constexpr unsigned StackSize  = 3500;
+	static constexpr unsigned StackSize  = 6000;
```

Comment:
Based on my tests a minimum of 4500 bytes is required. I'll check again once @pavel-kirienko's changes have landed and submit a PR with the actual size required.